openlibrary/core/lending.py

```diff
@@ -211,7 +216,14 @@ def get_available(limit=None, page=1, subject=None, query=None,
     """
     url = url or compose_ia_url(
         limit=limit, page=page, subject=subject, query=query,
-        work_id=work_id, _type=_type, sorts=sorts)
+        work_id=work_id, _type=_type, sorts=sorts
+    )
+    if not url:
+        fmt = (
+            "get_available(limit={}, page={}, subject={}, query={}, "
+            "work_id={}, _type={}, sorts={}"
+        )
+        logger.error(fmt.format(limit, page, subject, query, work_id, _type, sorts))
```

Comment:
In theory, it's faster to have `logger` be the one to format a string, but it doesn't matter.

Great that we're logging this now.

Reply:
Interesting! I didn't know you could do that :)